### PR TITLE
Mc 18215 add routes documentation

### PR DESCRIPTION
**Pull request by Vioarr**
#### Changes made
Added documentation for creating a route, there are other changes to the pricing markdown file but those were added in ages ago, can't remove the commit.

#### Related PRs
[MC-18215](https://github.com/cloudops/cloudmc-aws-plugin/pull/308)

### CODE REVIEW

File: source/includes/aws/_routes.md [hunk 1]
```diff
@@ -1,12 +1,52 @@
...
+
+> Request body example for an instance with custom security group:
+
+```json
+{
+  "cidrBlock": "0.0.0.0/0",
+  "id": "0.0.0.0/0",
+  "routeTableId": "rtb-0c43fead9b67edc8b"
+}
+
```
Comment (joelplourde4) [inline source/includes/aws/_routes.md:27]:
```suggestion
```
```

Comment (Vioarr) [reply source/includes/aws/_routes.md:37]: 96698c4f5af26a2b0e74bbc5093bcd8bbeeb67da

File: source/includes/aws/_routes.md [hunk 2]
```diff
@@ -1,12 +1,62 @@
...
+```
+
+> Request body example for an instance with custom security group:
+
+```json
+{
+  "cidrBlock": "0.0.0.0/0",
+  "id": "0.0.0.0/0",
+  "routeTableId": "rtb-0c43fead9b67edc8b"
+}
```
Comment (joelplourde4) [inline source/includes/aws/_routes.md]: "id" should be removed
```suggestion
{
  "cidrBlock": "0.0.0.0/0",
  "routeTableId": "rtb-0c43fead9b67edc8b"
}
```

**Review by joelplourde4 — COMMENTED** [review]
Thank you for including example of possible routes

File: source/includes/aws/_routes.md [hunk 3]
```diff
@@ -1,12 +1,62 @@
...
+```
+
+<code>POST /services/<a href="#administration-service-connections">:service_code</a>/<a href="#administration-environments">:environment_name</a>/routes</code>
+
+Create a new route in a route table, must have an attached internet gateway to perform the operation.
+
+| Required     | &nbsp;                                                                                                                                                                           |
+| ------------ | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| cidrBlock    | IPv4 range of the traffic you want to direct. Malformed IPv4 addresses, addresses within within AWS’s link-local address space, and CIDR blocks with invalid masks are rejected. |
+| id           | Auto-generated ID used for internal use, same as the CIDR block.                                                                                                                  |
```
Comment (emilyroser) [inline source/includes/aws/_routes.md]: remove id here too

**Review by emilyroser — APPROVED** [review]
besides changes ^ lgtm!

**Pull request merged**